bigtop-packages/src/rpm/ranger/SPECS/ranger.spec

```diff
@@ -120,7 +120,10 @@ Requires: coreutils, insserv
 
 # CentOS 5 does not have any dist macro
 # So I will suppose anything that is not Mageia or a SUSE will be a RHEL/CentOS/Fedora
-%if %{!?suse_version:1}0 && %{!?mgaversion:1}0
+# using the openeuler-lsb replace redhat-lsb in openEuler
+%if 0%{?openEuler}
+Requires: coreutils, openeuler-lsb
+%elif %{!?suse_version:1}0 && %{!?mgaversion:1}0
```

Comment:
Could you please kindly build and run smoke tests of Ranger on Fedora-36 or Rockylinux to double check the `if branch` ?
Thanks.

Reply:
ok, i will verfiy it on Fedora-36 and Rockylinux.

Reply:
1. I got a error using **ranger 2.4.0**  when smoke test in fedora-36/Rockylinux/openEuler, the error log is:
![image](https://github.com/apache/bigtop/assets/11262916/7a4b7e8e-0c80-4216-8e5f-59ea70fc13ea)
![image](https://github.com/apache/bigtop/assets/11262916/c906aff6-2775-4502-b811-bbc91582a1b6)

2. The cause is python3 support, the code is:
ranger-release-ranger-2.4.0\security-admin\scripts\db_setup.py
![image](https://github.com/apache/bigtop/assets/11262916/9e265242-535c-42b7-8c8d-96f29dec1ee5)

  Ranger fixed the bug by the commit, and ranger master branch toke it.
  https://github.com/apache/ranger/commit/f6b69ca5846de9a7dfced70ae8e403388ecb89fe

3. With the commit https://github.com/apache/ranger/commit/f6b69ca5846de9a7dfced70ae8e403388ecb89fe, smoke test is successful in openEuler/Fedora-36/Rockylinux, the log is:

**_openEuler smoketest ok_**
![image](https://github.com/apache/bigtop/assets/11262916/8edebf46-5a6d-4c0c-8003-c9a25ba21846)

**_Fedora-36 smoketest ok_**
![image](https://github.com/apache/bigtop/assets/11262916/79677593-0172-4be6-997a-85541d3c2fd0)

**_Rockylinux smoketest ok_**
![image](https://github.com/apache/bigtop/assets/11262916/120b8fe6-0c06-4b87-b80c-18046f123cc6)

4. The patch we can push it after you review and agree.
![image](https://github.com/apache/bigtop/assets/11262916/4b2aca3a-5530-48dc-8c3d-e5b16312d2fe)

@guyuqi

Reply:
Yes, the issues could be reproduced in the other Distros.
I would like to cherry pick the [relevant commit ](https://github.com/apache/ranger/commit/f6b69ca5846de9a7dfced70ae8e403388ecb89fe) to master branch.
Thanks for your testing and reporting it.